<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageClasses</class>
 <widget class="QWidget" name="ManageClasses">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tWManage">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Add Grade</string>
    </attribute>
    <widget class="QLabel" name="lGradeName">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>65</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Grade Name :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lEGradeName">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pBAddGrade">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>62</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Grade</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pBClose">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>192</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Add Class</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Class Name :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lEClassName">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Students :</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="tEStudents">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>461</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pBCreateClass">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>80</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Create class</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pBClose_2">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>202</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Add Student</string>
    </attribute>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Class</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cBSelectClass">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Student Index</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lEStudentName">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pBAddStudent">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>60</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Student</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pBClose_3">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>190</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Remove Grade</string>
    </attribute>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Grade</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cBSelectGrade">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pBRemoveGrade">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>60</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove Grade</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pBClose_4">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>192</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Remove Class</string>
    </attribute>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Class</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cBSelectClass_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pBRemoveClass">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>50</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove Class</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pBClose_5">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>192</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Remove Student</string>
    </attribute>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Class</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cBSelectClass_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>125</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Student</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cBSelectStudent">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>120</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pBRemoveStudent">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>70</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove Stuent</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pBClose_6">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>192</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pBClose</sender>
   <signal>clicked()</signal>
   <receiver>ManageClasses</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>655</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>843</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBClose_2</sender>
   <signal>clicked()</signal>
   <receiver>ManageClasses</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>655</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>805</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBClose_4</sender>
   <signal>clicked()</signal>
   <receiver>ManageClasses</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>653</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>742</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBClose_3</sender>
   <signal>clicked()</signal>
   <receiver>ManageClasses</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>645</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>704</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBClose_5</sender>
   <signal>clicked()</signal>
   <receiver>ManageClasses</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>647</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>801</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBClose_6</sender>
   <signal>clicked()</signal>
   <receiver>ManageClasses</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>643</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>661</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
